regression/contracts/assigns_enforce_statics_subfunctions/test2.desc

```diff
@@ -0,0 +1,25 @@
+CORE
+test2.c
+--enforce-contract foo _ --nondet-static
+^EXIT=10$
+^SIGNAL=0$
+^\[bar\.\d+\] line 11 Check that tmp is assignable: SUCCESS$
+^\[bar\.\d+\] line 12 Check that \*tmp is assignable: SUCCESS$
+^\[bar\.\d+\] line 17 Check that i is assignable: SUCCESS$
+^\[bar\.assertion.\d+\] line 21 i == tmp expecting FAILURE: FAILURE$
+^\[bar\.\d+\] line 26 Check that \*i is assignable: FAILURE$
+^\[bar\.assertion\.\d+\] line 30 \*i in range expecting FAILURE: FAILURE$
+^\[foo\.\d+\] line 39 Check that y is assignable: SUCCESS$
+^\[foo\.\d+\] line 41 Check that \*y is assignable: SUCCESS$
+^\[main\.assertion\.\d+] line 50 0 == x expecting FAILURE: FAILURE$
+^\[main\.assertion\.\d+] line 54 old_x == x expecting FAILURE: FAILURE$
+^VERIFICATION FAILED$
+--
+^\[main\.assertion\.\d+] line 50 0 == x expecting FAILURE: SUCCESS$
+^\[main\.assertion\.\d+] line 54 old_x == x expecting FAILURE: SUCCESS$
+--
+Checks whether static local and global variables are correctly tracked
+in assigns clause verification, across subfunction calls.
+TODO add these lines once havocing of static pointers is fixed
+^\[bar\.assertion\.\d+\] line 21 i == tmp expecting FAILURE: SUCCESS$
+^\[bar\.assertion\.\d+\] line 30 \*i in range expecting FAILURE: SUCCESS$
```

Comment:
Are we tracking this request on a GitHub issue? We should add the link here if yes.